airflow/www/templates/airflow/xcom.html

```diff
@@ -0,0 +1,24 @@
+{% extends "airflow/task_instance.html" %}
+{% block title %}Airflow - DAGs{% endblock %}
+
+{% block body %}
+    {{ super() }}
+    <h4>{{ title }}</h4>
+    <div>
+
+        <h5>Variables</h5>
```

Comment:
You may not need the "Variables" subheading here - does it add anything?

Also "Variables" also refers to Airflow Variables.